include/boost/detail/winapi/jobs.hpp

```diff
@@ -79,24 +92,20 @@ const DWORD_ JOB_OBJECT_ALL_ACCESS_ = (STANDARD_RIGHTS_REQUIRED_ | SYNCHRONIZE_
 #if !defined( BOOST_NO_ANSI_APIS )
 using ::OpenJobObjectA;
 #endif
-
 using ::OpenJobObjectW;
-
 using ::AssignProcessToJobObject;
-#if BOOST_USE_WINAPI_VERSION >= BOOST_WINAPI_VERSION_WINXP
 using ::IsProcessInJob;
-#endif
 using ::TerminateJobObject;
 
 #if !defined( BOOST_NO_ANSI_APIS )
 BOOST_FORCEINLINE HANDLE_ CreateJobObjectA(LPSECURITY_ATTRIBUTES_ lpJobAttributes, LPCSTR_ lpName)
 {
-    return ::CreateJobObjectA(reinterpret_cast< ::_SECURITY_ATTRIBUTES* >(lpJobAttributes), lpName);
+    return ::CreateJobObjectA(winapi::detail::cast_ptr(lpJobAttributes), lpName);
```

Comment:
What are these changes for? `cast_ptr` is not intended to be a replacement for `reinterpret_cast`.

Reply:
I can remove them.  I was trying to normalize on patterns I saw elsewhere.